testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/microprofile/opentracing/SimpleRestClientTestCase.java

```diff
@@ -0,0 +1,89 @@
+package org.jboss.as.test.integration.microprofile.opentracing;
```

Comment:
For consistency this should have a copyright header.

Reply:
None of the tests on this package have it. Should I apply it to all of them as part of this PR, or should I open a new PR?

Reply:
Ah nah that's fine then. We can do that later anyway :)